presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -923,7 +924,7 @@ public ConnectorTableHandle createTemporaryTable(ConnectorSession session, List<
                 buildInitialPrivilegeSet(table.getOwner()),
                 Optional.empty(),
                 false,
-                new PartitionStatistics(createEmptyStatistics(), ImmutableMap.of()));
+                new PartitionStatistics(createZeroStatistics(), columnHandles.stream().collect(toImmutableMap(HiveColumnHandle::getName, x -> empty()))));
```

Comment:
You can use https://github.com/prestodb/presto/blob/master/presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/Statistics.java#L244 to create empty statistics.